tests/bdd/tokens/test_generate.py

```diff
@@ -0,0 +1,169 @@
+# SPDX-FileCopyrightText: 2022-2023 VMware Inc
+#
+# SPDX-License-Identifier: MIT
+import ast
+
+from pytest_bdd import given, scenario, then, when
+from pytest_bdd.parsers import parse
+
+
+@scenario(
+    "../features/tokens/generate.feature",
+    "Admin uses HTTP API to generate a token",
+)
+def test_admin_uses_http_api_to_generate_a_token():
+    """Admin uses HTTP API to generate a token."""
+
+
+@given("the admin has the admin password", target_fixture="admin_passwd")
+def the_admin_has_the_admin_password(get_admin_pwd):
+    return get_admin_pwd
+
+
+@given(
+    "the admin gets an 'access_token' by logging in to '/api/v1/token' with a "
+    "'write:token' scope",
+    target_fixture="access_token",
+)
+def the_admin_has_generated_an_access_token_with_write_token(access_token):
+    return access_token
+
+
+@given(
+    parse("the admin adds Authorization Bearer {token} in the 'headers'"),
```

Comment:
I think this should be changed to:
``` suggestion
    parse("the admin adds Authorization Bearer 'access_token' in the 'headers'"),
```

Reply:
This works for now because of the reason described in @KAUTH comment here: https://github.com/repository-service-tuf/repository-service-tuf-api/pull/369#discussion_r1242615306